numpy_ringbuffer/__init__.py

```diff
@@ -168,7 +168,11 @@ def __getitem__(self, item):
 		if not isinstance(item, tuple):
 			item_arr = np.asarray(item)
 			if issubclass(item_arr.dtype.type, np.integer):
-				item_arr = (item_arr + self._left_index) % self._capacity
+				if self.is_full:
+					item_arr = (item_arr + self._left_index) % self._capacity
+				else:
+					item_arr = ((item_arr + self._left_index)
+								% self._right_index)
```

Comment:
This looks wrong to me for cases when `left_index != 0`

Reply:
`test_append` causes the buffer to wrap, which makes `self._left_index` be greater than 0, and it passes